lte/gateway/c/oai/tasks/nas/emm/TrackingAreaUpdate.c

```diff
@@ -802,7 +805,7 @@ static int _emm_tracking_area_update_accept(
 
         OAILOG_INFO(
             LOG_NAS_EMM,
-            "EMM-PROC  - Timer T3450 (%ld) expires in %ld"
+            "EMM-PROC  - Timer T3450 (%ld) expires in %d"
```

Comment:
As timer.sec has changed from "long" to "uint32_t", the printf specifier would change from %ld to %lu .